velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -218,10 +221,22 @@ dwio::common::StripeProgress getStripeProgress(
  * This method assumes each input `ColumnarBatch` have same schema.
  */
 void Writer::write(const VectorPtr& data) {
+  auto rowVector =
+      CHECK_NOTNULL(std::dynamic_pointer_cast<const velox::RowVector>(data));
+  auto rowType =
+      CHECK_NOTNULL(std::dynamic_pointer_cast<const velox::RowType>(schema_));
+
+  auto renamedVector = std::make_shared<facebook::velox::RowVector>(
```

Comment:
@majetideepak Updated the type in writer method. Can you help to review again? Thanks.